spec/features/user_profiles_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'helpers'
+
+feature 'user can view profile' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'users can view their profile' do
+    sign_in_as(user)
+    make_review_for_game(game)
+    visit users_path
+    within(".column") do
+      click_link user.username
+    end
+    expect(page).to have_content(user.username)
+    expect(page).to have_content("review of #{game.name}")
+  end
```

Comment:
Simplify. Testing shows expectations for specific scenarios. If you're scenario is a user can visit their own profile, then the test would be this:

```ruby
sign_in_as(user)
visit user_path(user)
expect(page).to have_content(user.username)
```

You don't need to check anything review related unless your scenario had to do with that (like 'see review of user' or something).